### PR TITLE
Api Catalogue sort with i18n support

### PR DESCRIPTION
**Pull request by manzapanza**
Closes: #1303

### CODE REVIEW

**Comment by bajiat** [comment]
@marla-singer Can I assign you as reviewer? 

@manzapanza Is this work in progress or ready for review?

**Comment by manzapanza** [comment]
@bajiat is ready for review

**Comment by marla-singer** [comment]
@manzapanza Add `local-compare-polyfill` in package.js as depence

**Comment by marla-singer** [comment]
@bajiat @brylie How can I check that sort feature works as expected?

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "jquery": "^2.2.4",
     "jquery-bbq": "^1.0.0",
     "jquery.rateit": "^1.0.23",
+    "locale-compare-polyfill": "0.0.2",
```
Comment (manzapanza) [inline package.json]: @marla-singer Is not this?

Comment (marla-singer) [inline package.json]: Exactly. Change order, add it after `js-yaml`

**Comment by manzapanza** [comment]
@marla-singer I ordered all packages by name

**Comment by bajiat** [comment]
@marla-singer 

One way to check is to see that if the sorting language is English, Ä is equivalent to A and Ö is equivalent to O (accents are ignored), while if Finnish is the sorting language, Ä is after Z and Ö is after Ä.

# English
Aid
Äiteen API
Awesome API
....
ZED's API

# Finnish
Aid
Awesome API
....
ZED's API
Äiteen API

**Comment by marla-singer** [comment]
@bajiat Thanks for clarification. So it works as expected for English and Finnish

**Pull request merged**